tests/test_ssh2.c

```diff
@@ -63,6 +92,24 @@ int main(int argc, char *argv[])
     }
 #endif
 
+#if defined(HAVE_PTHREAD_H)
+    pthread_t thread;
+
+    struct sigaction signal_handler_config = {
+        .sa_flags = SA_RESTART | SA_SIGINFO,
+        .sa_sigaction = signal_handler_function
+    };
+    sigemptyset(&signal_handler_config.sa_mask);
```

Comment:
`sigemptyset` is non-portable.

Reply:
Also OK to be non-portable for the same reason as below